janusgraph-lucene/src/main/java/org/janusgraph/diskstorage/lucene/NumericTranslationQueryParser.java

```diff
@@ -67,21 +73,21 @@ protected Query getRangeQuery(final String field, final String start, final Stri
     }
 
     @Override
-    protected Query newTermQuery(Term t) {
```

Comment:
Why you removed `newTermQuery`?

Reply:
By call hierarchy newTermQuery invoked from newFieldQuery